okhttp/src/main/java/com/squareup/okhttp/Dns.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2012 Square, Inc.
```

Comment:
It's a shame you can't force git to honor a rename + change. I assume this started its life as a `Network.java` rename, hence the 2012.